lib/reline/windows.rb

```diff
@@ -249,9 +251,15 @@ def self.scroll_down(val)
   end
 
   def self.clear_screen
-    # TODO: Use FillConsoleOutputCharacter and FillConsoleOutputAttribute
-    write "\e[2J"
-    write "\e[1;1H"
+    coordScreenTopLeft = 0
+    written = 0.chr * 4
+    csbi = 0.chr * 22
+    return if @@GetConsoleScreenBufferInfo.call(@@hConsoleHandle, csbi) == 0
+    dwConSize = csbi[0, 4].unpack('SS').inject(:*)
+    wAttributes = csbi[8, 2].unpack('S').first
+    @@FillConsoleOutputCharacter.call(@@hConsoleHandle, 0x20, dwConSize, coordScreenTopLeft, written)
+    @@FillConsoleOutputAttribute.call(@@hConsoleHandle, wAttributes, dwConSize, coordScreenTopLeft, written)
+    @@SetConsoleCursorPosition.call(@@hConsoleHandle, coordScreenTopLeft)
```

Comment:
Could you name all variables by snake case like `cood_screen_top_left`, `con_size`, and `attributes`.